pkg/datapath/loader/netlink.go

```diff
@@ -131,26 +130,22 @@ func graftDatapath(ctx context.Context, mapPath, objPath, progSec string) error
 	return nil
 }
 
-// DeleteDatapath filter from the given ifName
-func (l *Loader) DeleteDatapath(ctx context.Context, ifName, direction string) error {
-	args := []string{"filter", "delete", "dev", ifName, direction}
-	cmd := exec.CommandContext(ctx, "tc", args...).WithFilters(libbpfFixupMsg)
-	_, err := cmd.CombinedOutput(log, true)
+func RemoveTCFilters(ifName string, tcDir uint32) error {
```

Comment:
Nit: maybe a small function comment saying "use `netlink.HANDLE_MIN_{INGRESS,EGRESS}` for as a tcDir param".